frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/types.ts

```diff
@@ -274,6 +278,11 @@ export type CommonData = {
       loadError: string;
       data: PersistentVolumeClaimKind[];
     };
+    [VMWizardProps.nadsData]?: {
```

Comment:
we should not overuse data portion of redux  and have just one source of data. We should deprecate this in the future.